app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -218,7 +227,7 @@ public void onFinish() {
   private boolean isExternalLinkPopup;
   private String documentParserJs;
   private DocumentParser documentParser;
-  private KiwixTextToSpeech tts;
+  public static KiwixTextToSpeech tts;
```

Comment:
Since all the private variables are declared in a sequence. You should declare this public variable where they are being declared.